gradle/groovyProject.gradle

```diff
@@ -46,7 +46,7 @@ testTasks.all { task ->
 jarTasks.all { jar ->
     jar.manifest.mainAttributes(
         (Attributes.Name.IMPLEMENTATION_TITLE.toString()): 'Gradle',
-        (Attributes.Name.IMPLEMENTATION_VERSION.toString()): version,
+        (Attributes.Name.IMPLEMENTATION_VERSION.toString()): versionBase
```

Comment:
AFAIK, we don't, but users might have. So I wouldn't remove it. Using `SPECIFICATION-VERSION` is not correct either, because our spec and implementation change hand-in-hand.

I think it should be fine to use `IMPLEMENTATION_VERSION` with the version base, but we should document this change somewhere. Release notes seems like the right place I guess.